src/main.ts

```diff
@@ -67,10 +104,37 @@ async function run(): Promise<void> {
       core.info(`Calling s3Update with: ${JSON.stringify(options)}`);
 
       await s3Update(options);
+
+      const logUrl = core.getInput("deployRunUrl").replace(/__deployPath__/, deployPath);
```

Comment:
This should handle the case where the `deployRunUrl` isn't specified. Otherwise when we release this new version of the action, all of the existing repos that use this action will break here.